modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/expression/FilterByCountIndividualSegmentsExpressionVisitorImpl.java

```diff
@@ -214,29 +255,43 @@ public JSONObject toJSONObject() {
 
 		public static class Day {
 
-			public Day(String operator, String value) {
+			public Day(String operator, List<String> value) {
```

Comment:
Do we need a list or could we use an array instead an define the constructor as String .. values?